sale_automatic_workflow/models/stock_picking.py

```diff
@@ -24,13 +24,13 @@ def validate_picking(self):
                 rounding = move.product_id.uom_id.rounding
                 if (
                     float_compare(
-                        move.quantity_done,
+                        move.quantity,
```

Comment:
Why changing this? The field has been renamed?

Reply:
Yes, see https://www.youtube.com/watch?v=KNcK-TPTpRg&t=631s

Reply:
In the slide, it seems `quantity_done` is now `picked`, not `quantity`, isn't it?

Reply:
Then you should also check if `picked` is True.

Reply:
@sanderlienaerts Is this applicable ?

Reply:
hi @rousseldenis, the condition has been pushed to the latest commit